tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/workspace/ProductVersionPossibleValuesService.java

```diff
@@ -64,9 +65,20 @@ protected void compute(Set<String> values) {
 
 		if (get(op.getShowAllVersionProduct())) {
 			values.addAll(_productVersions);
+			String[] productVersions = _productVersions.toArray(new String[0]);
+
+			if (!ListUtil.isEmpty(productVersions)) {
+				op.setProductVersion(productVersions[0]);
```

Comment:
The possible value service should not set a value.  That should be done by default value service.

Reply:
yes, but we remove defautl value and this will not cause sapphire refresh more time. it will be empty when user firstly open wizard.